2021-07-27 19:35:06,893 - get_json_data.py[line:47] - WARNING: get json data failed: ajax_url = http://data.champdas.com/getMatchDefencesRateAjax.html, match_id = 14946, half = 1
2021-07-27 19:35:06,893 - get_json_data.py[line:48] - WARNING: HTTPConnectionPool(host='data.champdas.com', port=80): Max retries exceeded with url: /getMatchDefencesRateAjax.html (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002977947A708>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
2021-07-27 19:35:50,100 - get_json_data.py[line:47] - WARNING: get json data failed: ajax_url = http://data.champdas.com/getMatchDefencesRateAjax.html, match_id = 14992, half = 0
2021-07-27 19:35:50,100 - get_json_data.py[line:48] - WARNING: HTTPConnectionPool(host='data.champdas.com', port=80): Max retries exceeded with url: /getMatchDefencesRateAjax.html (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000029777D52D08>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
